tools/migrate_pmid_to_uid/migrate_pmid_to_uid.go

```diff
@@ -0,0 +1,366 @@
+package main
+
+/* CHECKLIST
```

Comment:
I'll leave it as-is. Probably doesn't belong in this file, but all of the files probably should be rev'd to remove the checklist.